app/konnect-platform/kong-identity.md

```diff
@@ -0,0 +1,154 @@
+* **Clients:** Represent machines that request tokens, such as microservices, mobile apps, or automation scripts.
+* **Scopes:** Define what those clients are allowed to access. 
+* **Claims:** Optional pieces of metadata, like user roles or environment tags, that can be included in tokens and forwarded to upstream services.
+
+To use Kong Identity for authentication, you must configure one of the supported plugins (OpenID Connect, OAuth2.0 Introspection, or Upstream OAuth). These plugins determine how tokens are validated, introspected, or passed along to upstream services.
+
+## Kong Identity client credential authentication flow
+
+The following diagram shows how authentication works with Kong Identity:
+
+<!--vale off-->
+{% mermaid %}
+sequenceDiagram
+    autonumber
+    participant client as Client Application
+    participant idsvc as Kong Identity Service
+    participant gateway as Kong Gateway
+    participant api as Customer API
+    client->>idsvc: request access token<br>(Client ID, Client Secret, requested scope)
+    idsvc->>idsvc: validate credentials
+    idsvc-->>client: access token, granted scope, expiry time
+    client->>gateway: /get_protected_resource<br>(access token, granted scope)
+    gateway->>idsvc: validate access token
+    idsvc-->>gateway: validate access token
+    gateway->>api: get_resource
+    api-->>gateway: resource
+    gateway-->>client: resource
+{% endmermaid %}
+<!--vale on-->
+
+## Kong Identity authorization code flow
+
+In the authorization code flow:
+1. Optional: The client application displays the user consent page and authenticates the user (this part is handled outside {{site.base_gateway}}). When the user clicks **Authorize**, the client app calls the `/authorize` endpoint created by attaching the OAuth2 plugin to a service.
+   {:.info}
+   > If an app requires user authentication, the authorization step must happen outside of {{site.konnect_short_name}}.
+3. The request includes the client ID, secret, and scopes the user consented to.
+4. The authorization server ({{site.base_gateway}} with OAuth2 plugin) validates the client credentials and returns an authorization code.
+5. The client exchanges this code at the `/oauth/token` endpoint for access tokens.
+6. The access token is used to call protected APIs.
+
+<!--
+For Consumer Group-scoped plugins:
+- Create a consumer per client in the respective control plane.
+- No need to migrate the client credential to a consumer credential.
+- The OIDC plugin maps clients to consumers using claims.
+- Create the required consumer groups and apply the plugin at the consumer group scope.
+- Add each consumer to the appropriate consumer group in the control plane.
+-->
+
+## Dynamic claim templates
+
+You can configure dynamic custom claims with dynamic claim templating to generate claims during runtime. These JWT claim values can be rendered as multiple types: 
+* Strings 
+* Integers
+* Floats
+* Booleans
+* JSON object or arrays
```

Comment:
```suggestion
You can configure dynamic claim templates to generate custom claims during runtime. These JWT claim values can be rendered as any of the following types:
* Strings 
* Integers
* Floats
* Booleans
* JSON object or arrays
```
How's this rephrase of the first sentence? The previous was a bit overwhelming with repetition.